mautic/mautic
# Bug Fix: fixed error when importing leads with stage

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | n
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | 
| BC breaks? | n
| Deprecations? | n

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 

### Required
#### Description:
This PR fixes bug when importing leads with stage.

#### Steps to test this PR:
1. Try to reproduce bug before applying this PR
2. Apply this PR and follow sames steps to reproduce bug, leads should now be created with a stage. First it tries to identify a stage with name given in CSV file if it doesn't exist it creates a new stage. NOTE that the new stage created does not have a weight assigned. Also it applies a stage to identified leads as well as new leads.

### As applicable
#### Steps to reproduce the bug:
1. Try to import leads with CSV file
2. Select a column to be the stage and an error will show up
